openstack_controller/datadog_checks/openstack_controller/api.py

```diff
@@ -68,6 +69,10 @@ def __init__(self, logger, ssl_verify, proxy_config, endpoint, auth_token):
         self.auth_token = auth_token
         self.headers = {'X-Auth-Token': auth_token}
 
+    def get_endpoint(self):
```

Comment:
this is the very same method as `NeutronApi`, should we just move it up to `AbstractApi` to avoid duplication?

Reply:
Good catch

Reply:
This can go now